### PR TITLE
Minor say code cleanup

### PR DESCRIPTION
**Pull request by optimumtact**
Remove two unused procs and move another to the mob/living say where it
is used

\>be me, browsing unrelated code
\>see that comment, have to check
\>tips katana

### CODE REVIEW

File: code/modules/flufftext/TextFilters.dm [hunk 1]
```diff
@@ -1,64 +0,0 @@
...
-			if(2)
-				word = "[first_letter]-[first_letter]-[word]"
-			if(3)
-				word = "[first_letter]-[word]"
-
-		split_phrase[index] = word
-
-	return sanitize(jointext(split_phrase," "))
-
-/proc/Stagger(mob/M,d) //Technically not a filter, but it relates to drunkenness.
```
Comment (optimumtact) [inline code/modules/flufftext/TextFilters.dm]: taking the piss to be sure

Comment (optimumtact) [reply code/modules/flufftext/TextFilters.dm]: \>d

Comment (Qustinnus) [reply code/modules/flufftext/TextFilters.dm]: i wish a proc gave me the d

File: code/modules/mob/living/say.dm [hunk 2]
```diff
@@ -59,6 +59,29 @@ GLOBAL_LIST_INIT(department_radio_keys, list(
 	"÷" = "cords"
 ))
 
+/mob/living/proc/Ellipsis(original_msg, chance = 50, keep_words)
+    if(chance <= 0)
+        return "..."
+    if(chance >= 100)
+        return original_msg
```
Comment (SpaceManiac) [inline code/modules/mob/living/say.dm]: I know the code was like this before but this seems opposite what it should be. Below, `chance` increases ellipsis occurrence, but here it seems to decrease it.

**Comment by vuonojenmustaturska** [comment]
surprise, old code is bad, but it can be refactored in a separate PR if anyone cares

**Pull request merged**